Collectors/DetailedCollector.py

```diff
@@ -310,12 +340,6 @@ def process_gstream(self, gstream, addr, sid):
                      event["block_bypass_cache"] = event.pop("b_bypass")
                      event["site"] = site
                      event["vo"] = self.returnVO(event["file_path"])
-
-                     if(event["block_hit_cache"] >= event["size"]):
-                         event["cache_request"] = "total"
-                     else:
-                         event["cache_request"] = "partial"
-
```

Comment:
Was this removal intended?

Reply:
Yes, this analysis of the total or partial requests is not accurate as we thought